packages/addon/src/background.ts

```diff
@@ -188,8 +188,28 @@ async function openUnifiedPopup() {
 }
 
 // Handle all messages from popup.
-browser.runtime.onMessage.addListener((message) => {
+browser.runtime.onMessage.addListener(async (message) => {
   switch (message.type) {
+    case 'TB/PING':
+      console.log('[background] got the ping from the bridge')
+      console.log(message);
+      break;
+
+    case 'TB/OIDC_TOKEN':
+      console.log(`I can haz OIDC TOKEN?`);
+      console.log(message);
+      // TODO: Store message.token
+      break;
```

Comment:
The ping is optional, but the token handler should store `message.token` to storage.

Also, other values could be sent over from the `auth-store.ts` (if we want the username, etc.)